### PR TITLE
Refactor launch files to easily include other environments

### PR DESCRIPTION
**Pull request by oscar-lima**
Refactor the mobipick labs launch configuration to allow other repository to derive from it.

### CODE REVIEW

**Comment by alexander-sung** [comment]
Requires https://github.com/DFKI-NI/mobipick/pull/5. Don't forget to add a pull request description here or there, which includes the intention description.

@jarkenau, please check if the naming is intuitive to you. It seems `world_config` is the filename prefix of the launch file which spawns the objects at specific poses. `world` is the filename of the sdf file.

**Review by mintar — APPROVED** [review]
LGTM! Once DFKI-NI/mobipick#5 is merged, this one is good to go as well.

**Pull request merged**
